install/kubernetes/values.yaml

```diff
@@ -171,6 +171,11 @@ tetragonOperator:
     # -- Enables the PodInfo CRD and the controller that reconciles PodInfo
     # custom resources.
     enabled: false
+  # -- Enables the Tetragon Operator metrics.
+  metrics:
```

Comment:
Could you name this section `prometheus` instead? I agree `metrics` makes sense, however, we already have `tetragon.prometheus` section for the Tetragon agent metrics, and IMO it's wroth keeping them consistent.